library/ssl_tls13_client.c

```diff
@@ -1328,6 +1328,7 @@ static int ssl_tls13_finalize_server_hello( mbedtls_ssl_context *ssl )
 
     handshake->transform_handshake = transform_handshake;
     mbedtls_ssl_set_inbound_transform( ssl, transform_handshake );
+    mbedtls_ssl_set_outbound_transform( ssl, ssl->handshake->transform_handshake );
```

Comment:
This change need double confirm with @hanno-arm . It break many tests in `prototype` .

Reply:
I prefer remove this change from this PR

Reply:
I still think that we should do something in this PR as if we do not switch to the handshake key to send the client Finished message then this message is not encrypted as it should be and thus this PR is not correct.

Reply:
Add mbedtls_ssl_set_outbound_transform in ssl_tls1_3_process_server_finished() done.